sql/updates/world/master/2025_03_11_00_Darkshore.sql

```diff
@@ -0,0 +1,531 @@
+
+(32862, 0, 46318, 52, 1), -- Hellscream's Missive
+(32862, 0, 5523, 52, 0), -- Small Barnacled Clam
+(32862, 0, 2589, 31, 0), -- Linen Cloth
+(32862, 0, 2592, 25, 0), -- Wool Cloth
+(32862, 0, 1179, 3, 0), -- Ice Cold Milk
+(32862, 0, 858, 1.7, 0), -- Lesser Healing Potion
+(32862, 0, 2455, 0.9, 0), -- Minor Mana Potion
+
+(32899, 0, 5523, 52, 0), -- Small Barnacled Clam
+(32899, 0, 2589, 31, 0), -- Linen Cloth
+(32899, 0, 2592, 25, 0), -- Wool Cloth
+(32899, 0, 1179, 3, 0), -- Ice Cold Milk
+(32899, 0, 858, 1.7, 0), -- Lesser Healing Potion
+(32899, 0, 2455, 0.9, 0) -- Minor Mana Potion
+AS new_data
+ON DUPLICATE KEY UPDATE
+    `Chance` = new_data.`Chance`,
+    `QuestRequired` = new_data.`QuestRequired`;
+
+
+-- creature_template_difficulty
+
+INSERT INTO `creature_template_difficulty` (`Entry`, `DifficultyID`, `LootID`)
+VALUES
+(32935, 0, 32935),
+(33009, 0, 33009),
+(33905, 0, 33905),
+(33311, 0, 33311),
+(33083, 0, 33083),
+(32890, 0, 32890),
+(33127, 0, 33127),
+(34302, 0, 34302),
+(32997, 0, 32997),
+(34413, 0, 34413),
+(2237, 0, 2237),
+(2071, 0, 2071),
+(2070, 0, 2070),
+(34350, 0, 34350),
+(32928, 0, 32928)
+AS new_data
+ON DUPLICATE KEY UPDATE
+    `LootID` = new_data.`LootID`;
+
+INSERT INTO `creature_template_difficulty` (`Entry`, `DifficultyID`, `LootID`, `GoldMin`, `GoldMax`)
+VALUES
+(33277, 0, 33277, 8, 24),
+(33262, 0, 33262, 8, 24),
+(32861, 0, 32861, 8, 24),
+(33071, 0, 33071, 2, 24),
+(34046, 0, 34046, 2, 24),
+(34033, 0, 34033, 2, 24),
+(32863, 0, 32863, 2, 24),
+(32869, 0, 32869, 2, 24),
+(32970, 0, 32970, 2, 24),
+(34248, 0, 34248, 2, 24),
+(32859, 0, 32859, 2, 24),
+(33021, 0, 33021, 2, 24),
+(33022, 0, 33022, 2, 24),
+(33020, 0, 33020, 12, 35),
+(33180, 0, 33180, 2, 24),
+(33179, 0, 33179, 2, 24),
+(33206, 0, 33206, 10, 26),
+(33207, 0, 33207, 10, 26),
+(34304, 0, 34304, 10, 26),
+(34385, 0, 34385, 10, 26),
+(33057, 0, 33057, 10, 26),
+(32888, 0, 32888, 10, 26),
+(32989, 0, 32989, 10, 26),
+(33044, 0, 33044, 10, 26),
+(33043, 0, 33043, 10, 26),
+(34405, 0, 34405, 10, 26),
+(32996, 0, 32996, 10, 26),
+(33079, 0, 33079, 10, 26),
+(34339, 0, 34339, 10, 26),
+(32860, 0, 32860, 10, 26),
+(32868, 0, 32868, 10, 26),
+(33980, 0, 33980, 10, 26),
+(32862, 0, 32862, 10, 26),
+(32899, 0, 32899, 10, 26)
+AS new_data
+ON DUPLICATE KEY UPDATE
+    `LootID` = new_data.`LootID`,
+    `GoldMin` = new_data.`GoldMin`,
+    `GoldMax` = new_data.`GoldMax`;
+
+
+-- creature
+
+INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnDifficulties`, `phaseUseFlags`, `PhaseId`, `PhaseGroup`, `terrainSwapMap`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curHealthPct`, `MovementType`, `npcflag`, `unit_flags`, `unit_flags2`, `unit_flags3`, `ScriptName`, `StringId`, `VerifiedBuild`)
+VALUES (10800002, 32960, 1, 0, 0, '0', 0, 0, 0, -1, 0, 0, 7445.79, -276.892, 13.6994, 1.61637, 300, 0, 0, 100, 0, NULL, NULL, NULL, NULL, '', NULL, 0)
```

Comment:
Is this spawn from sniffs? You're missing creature_addon data (npc is using invis and feign death auras)